Package.swift

```diff
@@ -10,7 +10,7 @@ let package = Package(
         .library(name: "FluentSQLiteDriver", targets: ["FluentSQLiteDriver"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/vapor/fluent-kit.git", from: "1.0.0-rc.1"),
+        .package(url: "https://github.com/vapor/fluent-kit.git", .branch("prepared-migration-filtering")),
```

Comment:
```suggestion
        .package(url: "https://github.com/vapor/fluent-kit.git", .from: "1.0.0-rc.1"),
```
Make sure this gets reverted before merging.